lib/github_message_factory.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class GithubMessageFactory
```

Comment:
Would be nice to have a quick comment over what this factory is taking in and spitting out.

```
This factory is used to take in a JSON event from the GithubFirehose service and parse it into a GithubMessage object.
```